release.yaml

```diff
@@ -73,9 +73,11 @@ rancher-webhook:
 sriov:
 - 101.0.2+up0.1.0
 - 101.0.3+up0.1.0
+- 102.0.0+up0.1.0
```

Comment:
Do we intend to support multiple versions of this chart in Rancher v2.7.2? If not, we should only have the 102.x.y chart in the release.yaml and remove the earlier versions using `make remove`

Reply:
I don't think multiple versions will be needed, I removed them.